src/docs/development/platform-integration/add-to-app/performance.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Load sequence, performance and memory
+---
+
+This page serves to inform the reader of the breakdown of the steps involved
```

Comment:
did you mean referring to ourselves in the third person (I think that's not what you meant since your suggestion does it too) or referring to the reader in the third person?